include/votca/tools/colors.h

```diff
@@ -89,45 +89,45 @@ class ColorSchemeBase {
 };
 
 template <typename TColorScheme>
-class Color : public ColorSchemeBase {
+class Color final: public ColorSchemeBase {
```

Comment:
Are the color schemes used anywhere?

Reply:
yes, they are used in the property class, but I do not think we are using that feature anywhere in particular. So yes and no. 

I think we should remove this step by step.